fsh/ig-data/input/pagecontent/index.md

```diff
@@ -211,7 +212,7 @@ needed, this can be accomplished by having the server send "unsolicited"
 messages, i.e., messages with no `responseToMessageId`, after a client's initial
 request.
 
-### Influence the EHR UI: `ui.*` messageType
+### Influence the EHR UI: `messageType ui.*`
```

Comment:
The string `messageType ui` won't appear in code every, so I think it's clearer if we don't put that string in backticks.

Reply:
OK, I can fix that.